test/Identity.FunctionalTests/xunit.runner.json

```diff
@@ -0,0 +1,3 @@
+{
+  "shadowCopy": false
```

Comment:
Btw, do you actually need this? Presumably we only need this for the deps file but we're have precompiled views.

Reply:
MVC still needs to load deps, so I rather keep it here.